libs/spirit-islander/data-access/src/lib/app-state.service.ts

```diff
@@ -59,17 +55,25 @@ export class AppStateService {
     }
 
     const expansions = EXPANSIONS.slice();
```

Comment:
Ideally, the `Options` class would be the way for the broader app to access game options rather than touching the constants directly